src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -329,6 +329,8 @@ public void Push(Page page)
             Add(page);
             page.Navigator = this;
 
+            AddPageToList(page);
```

Comment:
It seems that this API shows content of current page.
If there are multiple pages, then there are push animation and then the pushed page is finally displayed.
Therefore, this API should be called in Finished callback.

Isn't it required to call this API in PushWithTransition() and Insert()?

Reply:
Okay, I added Add~ method(Currently, Show~) to PushWithTransition(), Push(), Insert(), PopWithTransition() for the new page, and Pop() for the new page.
Could you check it again?